src/Bynder/Api/Impl/PermanentTokens/RequestHandler.php

```diff
@@ -19,7 +19,12 @@ public function __construct($configuration)
 
     protected function sendAuthenticatedRequest($requestMethod, $uri, $options = [])
     {
-        $request = new \GuzzleHttp\Psr7\Request($requestMethod, $uri, $options);
+        $headers = array_filter($options, function ($key) {
```

Comment:
I did the same kind of filter for `form_params` in #65 
I guess there might be other key to remove here too..